src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -348,4 +390,78 @@ void getTxInfoToSign_whenTransactionReceiptNotFoundInSubsequentBlock_shouldThrow
             pegoutBtcTransaction
         ));
     }
+
+    @ParameterizedTest
+    @MethodSource("getTxInfoToSignArgProvider")
+    void getTxInfoToSign_whenTransactionReceiptHasPegoutTransactionCreatedEvent_returnOk(
+        int version,
+        byte[] serializedOutpointValues,
+        List<Coin> expectedOutpointValues
+    )
+        throws HSMReleaseCreationInformationException {
+        // Arrange
+        List<LogInfo> logs = new ArrayList<>();
+
+        CallTransaction.Function releaseRequestedEvent = BridgeEvents.RELEASE_REQUESTED.getEvent();
+        byte[] releaseRequestedSignatureTopic = releaseRequestedEvent.encodeSignatureLong();
+        List<DataWord> topics = new ArrayList<>();
+        topics.add(DataWord.valueOf(releaseRequestedSignatureTopic));
+        topics.add(DataWord.valueOf(pegoutCreationRskTx.getHash().getBytes()));
+        topics.add(DataWord.valueOf(pegoutBtcTx.getHash().getBytes()));
+        logs.add(new LogInfo(PrecompiledContracts.BRIDGE_ADDR.getBytes(), topics, null));
```

Comment:
Should we also add the non-indexed data? To make it as realistic as possible. We could have a utils class to create events given some data.

```
class EventsTestUtils {

     LogInfo createReleaseRequestedEvent(pegoutCreationRskTx, pegoutBtcTx, amount() { ..
```
what do you think?

Reply:
Agree. I pushed the changes with `TestUtil` class. Thanks.